### ISSUE TITLE
Line wrapping

### ISSUE THREAD

**Issue by afraser** [snookca/CoffeeConsole#12]
It's absurdly often that I find myself writing lines like this:

```
example_highlights = app.authorized.courses.reader.concept.content.get('atom').get('highlights').filter((h) => h.get('highlight_data')[0].highlight_type == 'example')
```

To make things a bit more frustrating, ⌘← doesn't bring my cursor to the beginning of the line so if I want to get to the beginning or end of the line it's pretty tedious.

**Comment by Catincan** [comment]
Bounty of $50.00 is available on this issue: https://www.catincan.com/bounty/https-github-com-snookca-coffeeconsole-issues-12

**Comment by michaelficarra** [comment]
I can't reproduce. All of Ctrl-a/Ctrl-e, ⌘←/⌘→, and Home/End work fine for me.

**Comment by MaffooBristol** [comment]
Also, why is the text so insanely huge!?

**Comment by afraser** [comment]
@MaffooBristol The text scales with the dev tools window. ⌘- will scale it down when the dev tools window is in focus. (Note: strangely ⌘- will also scale the web view if focused on coffeeconsole, so you'll want to click on the dev tools top bar, or hit esc to bring up the console and select that before scaling the dev tools text)

**Comment by MaffooBristol** [comment]
Yeah I noticed that thanks, but I don't want the whole dev tools panel to be scaled, just the default font to not be so large, or at least a setting to adjust it yourself!
